src/text/parsers/comments.rs

```diff
@@ -11,8 +11,7 @@ use nom::sequence::{delimited, preceded};
 pub(crate) fn whitespace_or_comments(input: &str) -> IonParseResult<&str> {
     recognize(many0_count(alt((
         // At least one character of whitespace...
-        multispace1,
-        // ...or a comment of any format.
+        whitespace, // ...or a comment of any format.
```

Comment:
:world_map: `multispace1` uses `nom`'s definition of whitespace. Now that the reader recognizes esoteric whitespace like vertical tab, this needs to use our custom `whitespace` parser.